README.md

```diff
@@ -311,6 +311,10 @@ flank:
 
   ## Local folder to store the test result. Folder is DELETED before each run to ensure only artifacts from the new run are saved.
   # local-result-dir: flank
+
+  ## Downloaded files preserves the original path of file. Required when file names are not unique.
+  ## Default: false  
+  # preserve-original-path: false
```

Comment:
Thanks for working on this! What do you think of `keep-file-path` for the key name?